roles/telemetry_verify_metrics/tasks/verify_mariadb_metrics.yml

```
@@ -7,7 +7,8 @@
       - kind: scrapeconfigs.monitoring.rhobs
         name: telemetry-mysqld-exporter
 
-- name: Get galera CR names
+- name: |
+    Get galera CR names
```

```suggestion
- name: Get galera CR names
```

Revert this part of the change